lib/htmldriver/htmldriver.md

```diff
@@ -122,7 +122,7 @@ r.out.ppm -q input=D_cell output=alien.ppm
 ppmquant 128 <alien.ppm |  ppmtogif >alien.gif
 
 # assemble some html with the image and the image map
-echo '<html><body><img src="alien.gif" usemap="#map">' >alien.html
+echo '<html><body>![alien](alien.gif)' >alien.html
```

Comment:
This seems wrong in the example. (It refers to options available in grass 5)

Reply:
I reverted this for now, maybe this entire thing can be removed